common/src/HttpMessageHelper.cs

```diff
@@ -5,7 +5,7 @@
 using System.Text;
 using Newtonsoft.Json;
 
-namespace Microsoft.Azure.Devices.Shared
```

Comment:
Getting rid of the last classes with Shared in the namespace. As they are internal, they can just live in the "root" of `Microsoft.Azure.Devices`, IMHO.